src/Symfony/Component/Validator/Resources/translations/validators.he.xlf

```diff
@@ -68,7 +68,7 @@
             </trans-unit>
             <trans-unit id="17">
                 <source>The mime type of the file is invalid ({{ type }}). Allowed mime types are {{ types }}.</source>
-                <target>סוג MIME של הקובץ אינו חוקי ({{ type }}). מותרים סוגי MIME {{ types }}.</target>
+                <target>סוג הקובץ אינו חוקי ({{ type }}). הסוגים המותרים הם {{ types }}.</target>
```

Comment:
If the term "MIME" is too technical for this message, it should be removed from the English version also. Otherwise I think it's better keeping the "MIME" term.

> The second option uses "סוג הקובץ" (file type), which is more commonly understood in Hebrew than the technical "MIME," making the message clearer while staying accurate.